providers/frame/polkadot_dev.ts

```diff
@@ -1,13 +1,24 @@
+import { File } from "../../codegen/frame/mod.ts"
 import * as $ from "../../deps/scale.ts"
 import { Env, PathInfo } from "../../server/mod.ts"
+import { fromPathInfo } from "../../server/PathInfo.ts"
 import { getAvailable } from "../../util/port.ts"
 import { FrameBinProvider } from "./FrameBinProvider.ts"
 
+const DEV_RUNTIME_PREFIXES = {
+  polkadot: 0,
+  kusama: 2,
+  westend: 42,
+  rococo: 42,
+} as const
+
 export interface PolkadotDevProviderProps {
   polkadotPath?: string
 }
 
 export class PolkadotDevProvider extends FrameBinProvider {
+  #userCount: Record<string, number> = {}
```

Comment:
Let's narrow the key, and IMO there's no need for this to be #private
```suggestion
  userCount: Record<DevRuntimeName, number | undefined> = {}
```

Reply:
Any suggestion to extract the runtime names from 
```
const $devRuntimeName = $.literalUnion(["polkadot", "kusama", "westend", "rococo"])
```
or should I just do
```
keyof typeof DEV_RUNTIME_PREFIXES
```